Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -367,9 +367,9 @@ PhysicalParticleContainer::AddPlasmaFromFile(const std::string s_f,
         std::shared_ptr<amrex::ParticleReal> ptr_uz = ps.second["momentum"]["z"].loadChunk<amrex::ParticleReal>();
         double const momentum_unit_z = ps.second["momentum"]["z"].unitSI();
 #   else
-        amrex::Abort("AddPlasmaFromFile is only implemented for 2D and 3D\n")
+        amrex::Abort("AddPlasmaFromFile is only implemented for 2D and 3D\n");
```

Comment:
Thanks for catching this!
And we will be able to remove this line after testing PR #956 in RZ :)

Reply:
I saw this and though: "that's probably a macro, no problem" xD Upsi!